apps/dash-3d-image-partitioning/plotly-common/dash_utils.py

```diff
@@ -0,0 +1,22 @@
+import dash
+import os
+
+
+def new_dash_app(f, assets_in_cwd=True, verbose=False, **dash_args):
```

Comment:
this function is not used (unless I'm missing something), can you please remove the file?